sdk/tests/types/input/utxo.rs

```diff
@@ -78,60 +72,6 @@ fn from_str() {
     );
 }
 
-#[test]
-fn dto_fields() {
-    let output_id = OutputId::from_str(OUTPUT_ID).unwrap();
-    let utxo_input = UtxoInput::from(output_id);
-    let utxo_dto = UtxoInputDto::from(&utxo_input);
-
-    assert_eq!(utxo_dto.kind, UtxoInput::KIND);
-    assert_eq!(utxo_dto.transaction_id, output_id.transaction_id().to_string());
-    assert_eq!(utxo_dto.transaction_output_index, output_id.index());
-
-    let input = Input::from(utxo_input);
-    let dto = InputDto::from(&input);
-
-    assert_eq!(dto, InputDto::Utxo(utxo_dto));
-}
-
-#[test]
-fn dto_roundtrip() {
```

Comment:
Can't we convert all these tests to a serde roundrip? The DTOs are not public but we should still test their implementation ?

Reply:
I'd rather just move these to the unit

Reply:
changed my mind